src/std/text.ab

```diff
@@ -504,3 +504,95 @@ pub fun rpad(text: Text, pad: Text, length: Int): Text {
 pub fun zfill(text: Text, length: Int): Text {
     return lpad(text, "0", length)
 }
+
+/// Counts the number of lines in the given text.
+///
+/// ### Usage
+/// ```ab
+/// let count = lines_count("one\ntwo\nthree")
+/// echo(count) // 3
+/// ```
+#[allow_absurd_cast]
+pub fun lines_count(text: Text): Int {
+    if text == "": return 0
+    return trust $ wc -l <<< "{text}" | tr -d ' ' $ as Int
+}
```

Comment:
I think that `count_lines` sounds better

```suggestion
/// ### Usage
/// ```ab
/// let count = count_lines("one\ntwo\nthree")
/// echo(count) // 3
/// ```
#[allow_absurd_cast]
pub fun count_lines(text: Text): Int {
    if text == "": return 0
    return trust $ wc -l <<< "{text}" | tr -d ' ' $ as Int
}
```